<launch>
    <arg name="rectification" default="true"/>
    <arg name="sonar_viewer" default="true"/>
    <arg name="rviz" default="true"/>
    <arg name="plot" default="true"/>
    <param name="/use_sim_time" value="true"/>

    <node if="$(arg sonar_viewer)" pkg="sonar_oculus" name="oculus_viewerM750d" type="oculus_viewer.py" output="screen" args="M750d"/>
    <node if="$(arg sonar_viewer)" pkg="sonar_oculus" name="oculus_viewerM1200d" type="oculus_viewer.py" output="screen" args="M1200d"/>

    <node if="$(arg rviz)" type="rviz" name="rviz" pkg="rviz" args="-d $(find sonar-camera-reconstruction)/rviz/merge.rviz" />

    <node pkg="tf2_ros" type="static_transform_publisher" name="world2baselink" args="0 0 0 0 0 0 map odom" />

    <include file="$(find bruce_slam)/launch/slam.launch">
    </include>

    <include file="$(find sonar-camera-reconstruction)/launch/merge_node.launch">
    </include>

</launch>